components/provisioning/org.wso2.carbon.identity.provisioning/src/main/java/org/wso2/carbon/identity/provisioning/dao/ProvisioningManagementDAO.java

```diff
@@ -298,15 +298,6 @@ public void updateProvisionedIdentifier(IdentityProvider newIdentityProvider,
             prepStmt = dbConnection.prepareStatement(sqlStmt);
             prepStmt.setInt(1, tenantId);
             prepStmt.setString(2, newIdentityProvider.getIdentityProviderName());
-            ResultSet rs = prepStmt.executeQuery();
```

Comment:
remove this code lines

 sqlStmt = IdPManagementConstants.SQLQueries.GET_IDP_BY_NAME_SQL;
            prepStmt = dbConnection.prepareStatement(sqlStmt);
            prepStmt.setInt(1, tenantId);
            prepStmt.setString(2, newIdentityProvider.getIdentityProviderName());
            ResultSet rs = prepStmt.executeQuery();

            if (rs.next()) {

                // id of the updated identity provider.
                int idpId = rs.getInt(1);

            }